src/GraphingInterfaces/IGraphRenderer.h

```diff
@@ -24,16 +24,14 @@ namespace Graphing::Renderer
 
 		virtual HRESULT DrawD2D1(ID2D1Factory* pDirect2dFactory, ID2D1RenderTarget* pRenderTarget, bool& hasSomeMissingDataOut) = 0;
         virtual HRESULT GetClosePointData(
```

Comment:
I getting a read access violation when hovering over the graph. I think you may need to update this and merge it with the latest changes, including your changes to the graph engine wrapper.

Reply:
pushed updated changes. let me know if you still hit this issue